projects/zeepr/index.js

```diff
@@ -0,0 +1,8 @@
+const { staking } = require("./helper/staking");
+
+module.exports = {
+  arbitrum: {
+    tvl: 0,
+    staking: staking("0xe46C5eA6Da584507eAF8dB2F3F57d7F578192e13"),
```

Comment:
no, this doesnt work, how do you expect us compute tvl, where staking contract is not provided, also, we still need a way to price the token

Reply:
If I only display staking, how should I write it? @g1nt0ki

Reply:
`staking(stakingAddress, tokenAddress)`